### DISCUSSION TITLE
Qiita表彰プログラムの受賞者はプロフィールページに何か情報があると良いかも

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#166] [要望・アイデア・不具合]
現状、Qiita表彰プログラムを受賞していてもプロフィールページには何も情報が表示されていません。
何かしらの情報が表示されているとユーザーも誇らしく思えるかもしれません。

- https://qiita.com/jnchito
- https://qiita.com/qiita-award/users

**Comment by kskwtnk** [comment]
@JunichiIto 
ご意見ありがとうございます！

こちらもう、なんというか全くもってその通りです 🙇 

現状、[最近リリースしたバッジ機能](https://qiita.com/release-notes#バッジ機能を追加しました)なども含めて、実績が他者から見えない箇所が多いです
プロフィールページのより良い姿を議論している最中でもありますので、議題として取り入れます！
